FacesofLondon/app/src/main/java/com/novoda/wallpaper/droidcon/TimeOfDayCalculator.java

```diff
@@ -0,0 +1,34 @@
+package com.novoda.wallpaper.droidcon;
+
+import java.util.Calendar;
+import java.util.Locale;
+
+import static com.novoda.wallpaper.droidcon.TimeOfDay.*;
+
+class TimeOfDayCalculator {
+
+    TimeOfDay currentTimeOfDay() {
+        Calendar calendar = Calendar.getInstance();
+        int hourOfDay = calendar.get(Calendar.HOUR_OF_DAY);
+        return forHourOfDay(hourOfDay);
+    }
+
+    TimeOfDay forHourOfDay(int hourOfDay) {
```

Comment:
Some more info on what do you mean by play with enums? I don't really get how making `hourOfDay` be a `TimeOfDay` would help considering we need to know which TimeOfDay it is